kolibri/core/assets/src/state/actions.js

```diff
@@ -685,6 +684,17 @@ function updateMasteryAttemptState(store, {
     });
 }
 
+function fetchPoints(store) {
+  if (store.state.core.session.user_id &&
+    store.state.core.session.kind[0] !== UserKinds.SUPERUSER) {
```

Comment:
in #1274 I tried to remove all kind-checking code like this and instead rely on `coreGetters.isSuperuser`